pom.xml

```diff
@@ -118,6 +118,20 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-source-plugin</artifactId>
             </plugin>
+
```

Comment:
https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.generic/blob/main/pom.xml#L84
. must be added to every pom otherwise we have a mess on sonarcloud